openslides/core/static/css/app.css

```diff
@@ -455,10 +455,34 @@ img {
     padding: 5px 15px;
 }
 
+.col2 .message .panel-heading {
+    padding: 3px 15px;
+}
+
 .col2 .message .panel-body {
     padding: 10px 15px;
 }
 
+.col2 .message .projektorbtn {
+    float: left;
+    width: auto;
+    margin: 5px 10px 5px 0px;
+}
+
+.col2 .message .innermessage {
+    float: left;
+    width: 180px;
+    max-width: 170px;
+    overflow: hidden;
+    white-space: nowrap;
```

Comment:
please remove this line. otherwise the message cannot break lines automatically. - I tested it successfuly without nowrap.